nmdc_schema/migrators/migrator_from_9_1_to_9_2.py

```diff
@@ -0,0 +1,43 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+from nmdc_schema.migrators.helpers import load_yaml_asset
+
+class Migrator_from_9_1_to_9_2(MigratorBase):
+    """previously: Migrates data from schema 9.1.0 to 9.2.0"""
```

Comment:
I think the `previously: ` prefix is something @turbomam added to a couple existing classes' docstrings when he removed the "patch" number (i.e. `A_B_C` --> `A_B`) from their names. I'd prefer it be omitted here.

```diff
- """previously: Migrates data from schema 9.1.0 to 9.2.0"""
+ """Migrates data from schema 9.1.0 to 9.2.0"""
```